enterprise-suite/console_test/args/args.go

```diff
@@ -0,0 +1,20 @@
+package args
```

Comment:
I don't think we need a separate package for this one file. What about sticking it in the `tests` package?

Reply:
I find it more readable to have commandline args in their own package, because any code that uses them can say `args.Namespace` to reference it. Can still change it if you feel strongly towards having less packages.

Reply:
Okay I'll let you decide since I don't feel that strongly. It just felt a little out of place at the top level to me. But I think it's fine and easy to move if we change our minds.